cstrml/tee/src/lib.rs

```diff
@@ -208,6 +208,11 @@ impl<T: Trait> Module<T> {
         }
     }
 
+    pub fn get_last_work_report(who: &T::AccountId) -> Option<WorkReport> {
```

Comment:
just rename to `get_work_report`? cause `current_report_slot` is like `private` attribute inside tee